posts/2017-10-15-creating-a-serverless-graphql-to-rest-gateway-on-top-of-the-flickr-api.md

```diff
@@ -370,11 +395,17 @@ module.exports = {
+There are three important things to note here.
+
+One: I specifically include the webpack config for this project highlight webpack's [provide plugin](https://webpack.js.org/plugins/provide-plugin/). It allows you to call exports from node modules without having to explicitly import them in the files in which you use them.
+
+This plugin saves me from having to remember to import frequently-used things such as the built-in GraphQL scalar types, which we'll be using often throughout this project. So, **if you are confused why you're seeing things like `GqlString` instead of `GraphQLString`, this is why.**
 
-Second of all, I want to point out that we're using `babel-plugin-transform-optional-chaining`, which adds support for the TC39 syntax proposal: [Optional Chaining](https://github.com/tc39/proposal-optional-chaining) aka the Existential Operator. You'll see this in the code base in the following format: `obj?.property` which is equivalent to `!!object.property ? object.property : undefined`. This syntactic sugar will test for the existence of a property or method before invocation, evaluating to undefined when a property or method does not exist. Using this syntax requires using [babel 7](https://babeljs.io/blog/2017/03/01/upgrade-to-babel-7), so keep that in mind before attempting to use the plugin in your own projects. It can be tricky to set up as babel 7 is still in beta and the syntax itself is a stage 1 proposal at the time of writing.
+Two: we're using `babel-plugin-transform-optional-chaining`, which adds support for the TC39 syntax proposal: [Optional Chaining](https://github.com/tc39/proposal-optional-chaining), aka the Existential Operator. You'll see this in the code base in the following format: `obj?.property` which is equivalent to `!!object.property ? object.property : undefined`.
 
-Finally take note that we're using a resolve alias, specifying `@` as the project root directory. This allows us to do project root relative imports, such as `import { invariant } from "@/utilities"`. Using this webpack feature is also a matter of personal preference, and I enjoy it as it helps with code organization and managing relative imports across refactors.
+This syntactic sugar will test for the existence of a property or method before invocation, evaluating to undefined when a property or method does not exist. Using this syntax requires using [babel 7](https://babeljs.io/blog/2017/03/01/upgrade-to-babel-7), so keep that in mind before attempting to use the plugin in your own projects. It can be tricky to set up as babel 7 is still in beta and the syntax itself is a stage 1 proposal at the time of writing.
```

Comment:
Need to add a note here to pin babel 7 versions to 7.0.0-beta.2 here, as over the weekend I learned the hard way that the latest version breaks everything.